[2024-03-21T20:23:31.457-0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: execute_python_operators2.task_a manual__2024-03-21T23:23:29.584276+00:00 [queued]>
[2024-03-21T20:23:31.461-0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: execute_python_operators2.task_a manual__2024-03-21T23:23:29.584276+00:00 [queued]>
[2024-03-21T20:23:31.461-0300] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2024-03-21T20:23:31.461-0300] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2024-03-21T20:23:31.461-0300] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2024-03-21T20:23:31.473-0300] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): task_a> on 2024-03-21 23:23:29.584276+00:00
[2024-03-21T20:23:31.474-0300] {standard_task_runner.py:55} INFO - Started process 6898 to run task
[2024-03-21T20:23:31.476-0300] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'execute_python_operators2', 'task_a', 'manual__2024-03-21T23:23:29.584276+00:00', '--job-id', '59', '--raw', '--subdir', 'DAGS_FOLDER/python_operators2.py', '--cfg-path', '/tmp/tmpbm_yhqfy']
[2024-03-21T20:23:31.477-0300] {standard_task_runner.py:83} INFO - Job 59: Subtask task_a
[2024-03-21T20:23:31.508-0300] {task_command.py:389} INFO - Running <TaskInstance: execute_python_operators2.task_a manual__2024-03-21T23:23:29.584276+00:00 [running]> on host DESKTOP-0DNP0LN.localdomain
[2024-03-21T20:23:31.556-0300] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=falcone
AIRFLOW_CTX_DAG_ID=execute_python_operators2
AIRFLOW_CTX_TASK_ID=task_a
AIRFLOW_CTX_EXECUTION_DATE=2024-03-21T23:23:29.584276+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-03-21T23:23:29.584276+00:00
[2024-03-21T20:23:31.557-0300] {logging_mixin.py:137} INFO - Task A executed!
[2024-03-21T20:23:31.557-0300] {python.py:177} INFO - Done. Returned value was: None
[2024-03-21T20:23:31.560-0300] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=execute_python_operators2, task_id=task_a, execution_date=20240321T232329, start_date=20240321T232331, end_date=20240321T232331
[2024-03-21T20:23:31.608-0300] {local_task_job.py:212} INFO - Task exited with return code 0
[2024-03-21T20:23:31.622-0300] {taskinstance.py:2596} INFO - 2 downstream tasks scheduled from follow-on schedule check
